### PR TITLE
Create io.github.FailurePoint.RandomNumberFive.json

### PR DESCRIPTION
**Pull request by FailurePoint**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: io.github.FailurePoint.RandomNumberFive.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "random-number-five",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "file:///home/ace/Projects"
```
Comment (hfiguiere) [inline io.github.FailurePoint.RandomNumberFive.json:33]: it won't find the source here.

Comment (FailurePoint) [reply io.github.FailurePoint.RandomNumberFive.json]: It has been fixed. Do I need to submit a new PR with the fix in it?

File: io.github.FailurePoint.RandomNumberFive.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "random-number-five",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/FailurePoint/RandomNumberFive.git"
```
Comment (hfiguiere) [inline io.github.FailurePoint.RandomNumberFive.json:33]: a commit or tag is required.

Comment (FailurePoint) [reply io.github.FailurePoint.RandomNumberFive.json:33]: How do I find that? I am still pretty new to the GitHub system.

File: io.github.FailurePoint.RandomNumberFive.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.FailurePoint.RandomNumberFive",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "random-number-five",
+    "finish-args" : [
+        "--share=network",
```
Comment (hfiguiere) [inline io.github.FailurePoint.RandomNumberFive.json:8]: why does a random number generator need the network?

Comment (FailurePoint) [reply io.github.FailurePoint.RandomNumberFive.json:8]: it was inserted by GNOME Builder by default, I thought I removed it but I had to rebuild the software a few so I must have missed it on this one.

**Comment by FailurePoint** [comment]
is this what you needed?

**Comment by hfiguiere** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

**Comment by FailurePoint** [comment]
What failed?
And why?

**Comment by FailurePoint** [comment]
I guess I don't understand...

File: io.github.FailurePoint.RandomNumberFive.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "modules" : [
+        {
+            "name" : "random-number-five",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/FailurePoint/RandomNumberFive.git",
+                    "commit" : "212f807"
```
Comment (hfiguiere) [inline io.github.FailurePoint.RandomNumberFive.json:33]: this is supposed to be the full sha

**Comment by FailurePoint** [comment]
It builds just fine on my laptop...

File: io.github.FailurePoint.RandomNumberFive.json (hunk 4 above)
Comment (FailurePoint) [reply io.github.FailurePoint.RandomNumberFive.json:33]: Good grief... where am I supposed to find that??

Comment (hfiguiere) [reply io.github.FailurePoint.RandomNumberFive.json:33]: also this one doesn't even exist

image

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

File: io.github.FailurePoint.RandomNumberFive.json (hunk 4 above)
Comment (FailurePoint) [reply io.github.FailurePoint.RandomNumberFive.json]: well where is the full sha??

Comment (FailurePoint) [reply io.github.FailurePoint.RandomNumberFive.json]: I don't understand where I am supposed to get that.

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

**Comment by FailurePoint** [comment]
What am I missing?
It has the full SHA now for a relevant commit and the correct URL...
I even removed the network requirement...
Why is it still failing build??

**Comment by hfiguiere** [comment]
It doesn't exist:

image

**Comment by FailurePoint** [comment]
why?? It shows up on my end:
Screenshot from 2023-07-05 18-22-36

**Comment by FailurePoint** [comment]
wait, are these changes supposed to be to the manifest in my upstream or the one in my pull request?

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

**Comment by FailurePoint** [comment]
OK. It is working now.

**Comment by FailurePoint** [comment]
@hfiguiere It seems to be working correctly. when you get a chance please review it. thanks!

**Comment by FailurePoint** [comment]
Hello? Is anyone looking at this anymore?

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

**Comment by FailurePoint** [comment]
bot, build io.github.FailurePoint.RandomNumberFive

File: io.github.FailurePoint.RandomNumberFive.json (hunk 1 above)
Comment (orowith2os) [reply io.github.FailurePoint.RandomNumberFive.json]: Just push it here.

File: io.github.FailurePoint.RandomNumberFive.json (hunk 2 above)
Comment (orowith2os) [reply io.github.FailurePoint.RandomNumberFive.json]: Get the commit from the version used; git will contain it, and GitHub will expose it in the UI.

**Comment by barthalion** [comment]
```
Warning: Failed to parse appdata: element 'content_rating' requires attribute 'type'
```

**Pull request closed without merging**